src/export/export-chakra.ts

```diff
@@ -117,64 +123,96 @@ const processTextStyles = (
 
   // Replace the font family in each text style with the corresponding token
   Object.keys(updatedTextStyles).forEach((key) => {
-    const value = updatedTextStyles[key];
-    const fontFamily = value.fontFamily;
-    if (typeof fontFamily === 'string') {
-      const token = getFontFamilyToken(fontFamily);
-      if (token) {
-        value.fontFamily = token;
+    if (typeof updatedTextStyles[key] !== 'string') {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      const value = updatedTextStyles[key];
+      const fontFamily = value.fontFamily;
+      if (typeof fontFamily === 'string') {
+        const token = getFontFamilyToken(fontFamily);
+        if (token) {
+          value.fontFamily = token;
+        }
       }
     }
   });
 
   return updatedTextStyles;
 };
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const reshapeDesignTokens = (tokens: any): ChakraTokens => {
+  const obj = tokens;
```

Comment:
If the goal of reassigning here is to avoid mutating the original object then we need to spread it. I also think we should rename the variable to `newTokens` to make it more explicit. e.g.

```
const newTokens = { ...tokens };
```

Reply:
🤦🏼‍♀️ omg of course, lol, thanks for reminding me how JS works 😂 
This likely explains my type issues in this comment https://github.com/PortableStudios/figma2theme/pull/352#discussion_r1022275855